packages/server/lib/screenshots.coffee

```diff
@@ -309,7 +309,10 @@ getPath = (data, ext, screenshotsFolder) ->
   if data.name
     names = data.name.split(pathSeparatorRe).map(sanitize)
   else
-    names = [data.titles.map(sanitize).join(RUNNABLE_SEPARATOR)]
+    ## they could have test titles that are not strings, 
+    ## remove those before trying to sanitize
+    stringTitles = _.filter(data.titles, _.isString)
```

Comment:
Maybe we should stringify the value instead of removing it? Any thoughts @brian-mann?

Reply:
Yeah, this is what @brian-mann said also. 😅 I will update.